install/prereqs.rst

```diff
@@ -0,0 +1,93 @@
+#############
+Prerequisites
+#############
+
+This page lists software needed to install and use the LSST Science Pipelines.
+
+.. _prereq-platforms:
+
+Platform compatibility
+======================
+
+The LSST Data Management reference platform is CentOS 7.
+This is the platform we officially develop, test, and operate with.
+
+Besides the reference platform, our developers and automatic tests regularly compile and run the Pipelines under CentOS 8, Debian and Ubuntu Linux, and macOS.
+See `LSST Stack Testing Status <https://ls.st/faq>`_ reports of building LSST software on various platforms.
+
+.. _system-prereqs:
+
+System prerequisites
+====================
+
+The Science Pipelines are developed and built using a standard `Conda`_ environment.
+This provides (almost; see below) all necessary tools and libraries for building and running the Pipelines.
+On installation, you will be given the option of automatically installing that environment.
+If you decline, your system must have all of the prerequisite listed below available:
+
+.. jinja:: default
+
+   - `macOS <https://github.com/lsst/scipipe_conda_env/blob/{{ scipipe_conda_ref }}/etc/conda3_packages-osx-64.yml>`_.
+   - `Linux <https://github.com/lsst/scipipe_conda_env/blob/{{ scipipe_conda_ref }}/etc/conda3_packages-linux-64.yml>`_.
+
+In addition to the Conda environment, the following provided by the host platform.
```

Comment:
```suggestion
In addition to the Conda environment, the following are provided by the host platform.
```

Reply:
Reworded to “the following packages must be installed on the host system”.